mcodingbot/tasks/stats.py

```diff
@@ -122,27 +121,32 @@ async def get_stats(bot: Bot) -> Stats:
     return _last_known_stats
 
 
+def format_float(n: int | float) -> int | float:
+    if float(n).is_integer():
+        return int(n)
+    return n
+
+
+def floor(number: int | float, ndigits: int = 0) -> float:
```

Comment:
This function does not floor. `truncate_decimals` is closer.

Reply:
Good point, thanks